fix_includes.py

```diff
@@ -968,24 +979,37 @@ class Myclass;
   start_line = 0
   while start_line < len(file_lines):
     line_info = file_lines[start_line]
-    if line_info.deleted or line_info.type != _NAMESPACE_START_RE:
+    if (line_info.deleted or 
+       (line_info.type != _NAMESPACE_START_RE and
+        line_info.type != _NAMESPACE_START_ALLMAN_RE)):
       start_line += 1
       continue
-    # Because multiple namespaces can be on one line
-    # ("namespace foo { namespace bar { ..."), we need to count.
-    # We use the max because line may have 0 '{'s if it's a macro.
-    # TODO(csilvers): ignore { in comments.
-    namespace_depth = max(line_info.line.count('{'), 1)
+    if line_info.type == _NAMESPACE_START_RE:
+      # Because multiple namespaces can be on one line
+      # ("namespace foo { namespace bar { ..."), we need to count.
+      # We use the max because line may have 0 '{'s if it's a macro.
+      # TODO(csilvers): ignore { in comments.
+      namespace_depth = max(line_info.line.count('{'), 1)
+    elif line_info.type == _NAMESPACE_START_ALLMAN_RE:
+      # For Allman namespaces, keep the start line and increment
+      # the namespace depths when the actual brace is encountered.
+      namespace_depth = 0
```

Comment:
It's a little disconcerting that `namespace_depth` is unbound if type is neither `_NAMESPACE_START_RE` nor `_NAMESPACE_START_ALLMAN_RE`. After some scanning, I found that that's guaranteed by the conditional above, but maybe it would be nice to add an `else` leg with an `assert False` to make sure that invariant doesn't break.

Reply:
@kimgr great point.  I've added an assert to guard against future additions.

Reply:
:+1: